.circleci/config.yml

```diff
@@ -1,318 +1,113 @@
 version: 2.1
 
 orbs:
-  build-tools: circleci/build-tools@2.6.2
-  jq: circleci/jq@volatile
-  node: circleci/node@dev:alpha
-  orb-tools: circleci/orb-tools@8.27.3
+  node: circleci/node@<<pipeline.parameters.dev-orb-version>>
+  orb-tools: circleci/orb-tools@9.0
+
+# Pipeline parameters
+parameters:
+  # These pipeline parameters are required by the "trigger-integration-tests-workflow"
+  # job, by default.
+  run-integration-tests:
+    type: boolean
+    default: false
+  dev-orb-version:
+    type: string
+    default: "dev:alpha"
+
+integration-tests: &integration-tests
+  [
+    integration-test-install-linux,
+    integration-test-install-mac,
+    node-test-job
+  ]
 
 jobs:
-  test:
-    parameters:
-      executor:
-        type: executor
-        default: node/default
-
-      node-version:
-        type: string
-        default: latest
-
-      node-alpine-version:
-        type: string
-        default: nodejs-current
-
-      npm-version:
-        type: string
-        default: latest
-
-      yarn-version:
-        type: string
-        default: ""
-
-    executor: <<parameters.executor>>
+# Install NodeJS into a non-node container.
+  integration-test-install-linux:
+    docker:
+      - image: cimg/base:stable
     steps:
       - checkout
-      - build-tools/install-ci-tools
-
       - node/install:
-          node-version: <<parameters.node-version>>
-          node-alpine-version: <<parameters.node-alpine-version>>
-          npm-version: <<parameters.npm-version>>
-          yarn-version: <<parameters.yarn-version>>
-          debug: true
-
-# yaml anchor filters
-integration-dev_filters: &integration-dev_filters
-  branches:
-    ignore: /.*/
-  tags:
-    only: /integration-.*/
-
-integration-master_filters: &integration-master_filters
-  branches:
-    ignore: /.*/
-  tags:
-    only: /master-.*/
+          node-version: "13.11.0"
```

Comment:
Could we add a step to install `nvm` before the `node/install` command runs, (or maybe this test could be done in another job), to test the nvm removal part of the command?